src/components/JoinGroupForm/JoinGroupForm.tsx

```diff
@@ -70,10 +68,65 @@ export default function JoinGroupForm() {
 
   const handleClick = async (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
I think we need a name like 'handleGetClass' or something more descriptive for this function, as this handleClick is not in charge of every click

Reply:
I meant this for line 69. Even tho it was not a change for this commit